src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -2,36 +2,42 @@
   "iris.shaders.reloaded": "Shader neu geladen!",
   "iris.shaders.toggled": "Shader umgeschaltet auf: %s",
   "iris.keybind.reload": "Shader neu laden",
-  "iris.keybind.shaderPackSelection": "Shaderpaket-Auswahlbildschirm",
+  "iris.keybind.shaderPackSelection": "Shader-Auswahlbildschirm",
   "iris.keybind.toggleShaders": "Shader umschalten",
   "iris.keybinds": "Iris",
-  "iris.shaders.reloaded.failure": "Neuladen der Shader fehlgeschlagen! Grund: %s",
-  "iris.shaders.toggled.failure": "Umschalten der Shader fehlgeschlagen! Grund: %s",
+  "iris.shaders.reloaded.failure": "Shader konnte nicht neu geladen werden! Grund: %s",
+  "iris.shaders.toggled.failure": "Shader konnte nicht umgeschaltet werden! Grund: %s",
   "iris.sodium.failure.title": "Iris konnte nicht geladen werden!",
   "iris.sodium.failure.download": "Sodium herunterladen",
   "iris.sodium.failure.reason.notFound": "Iris erfordert Sodium für optimale Leistung, aber Sodium ist nicht installiert. Bitte lade Sodium herunter, füge es deinem Mods-Ordner hinzu und starte das Spiel neu.",
   "iris.sodium.failure.reason.incompatible": "Iris erfordert Sodium für optimale Leistung, aber es wurde eine inkompatible Version gefunden. Bitte lade die passende Version von Sodium herunter, füge sie deinem Mods-Ordner hinzu, entferne die falsche Version und starte das Spiel neu.",
 
   "options.iris.apply": "Anwenden",
   "options.iris.refresh": "Aktualisieren",
-  "options.iris.openShaderPackFolder": "Shaderpaket-Ordner öffnen ...",
-  "options.iris.shaderPackSettings": "Shaderpaket-Einstellungen ...",
-  "options.iris.shaderPackList": "Shaderpaket-Liste ...",
-  "options.iris.refreshShaderPacks": "Shaderpakete aktualisieren",
+  "options.iris.openShaderPackFolder": "Shader-Ordner öffnen ...",
+  "options.iris.shaderPackSettings": "Shader-Einstellungen ...",
+  "options.iris.shaderPackList": "Shader-Liste ...",
```

Comment:
Why is there a distinction between "shader" and "shaderpaket" here? Sometimes you write the one and sometimes you write the other. In the English original it's always "shaderpack" except for when "shaders" are referenced as a feature in general. (see the first few lines of the file) It's actually pretty consistent in the original I think.

Reply:
Because some of the strings don’t quite fit on the buttons and I‘ve adjusted them accordingly as well as those that follow a similar scheme

Reply:
Just looking it at, it feels weird, but if this is required to fit the long words into the buttons it's good.

Reply:
Yeah I know but it’s necessary (IMS allowed me to do that) and optifine as well as vanilla do the same (I really dislike the German translation for vanilla minecraft because it includes so many weird sounding words and way to many abbreviations)

Reply:
yeah the German translation for Minecraft feels somewhat clunky and unnatural at times. But I think at the same time that means the standard for how natural German translations have to sound isn't very high.